app/models/foi_attachment.rb

```diff
@@ -298,4 +295,7 @@ def text_type?
     AlaveteliTextMasker::TextMask.include?(content_type)
   end
 
+  def extractable_to_html?
+    %w[text/plain application/rtf].include?(content_type)
```

Comment:
Maybe not for now... it would be nice if these content types weren't hardcoded here as this could delegate to `AttachmentToHTML`. I believe this is `true` if `AttachmentToHTML.send(:adapter_for, foi_attachment) != AttachmentToHTML::Adapters::CouldNotConvert`

Aside: for PDF files this is `true` because `has_google_docs_viewe?` returns true, but we use `pdftohtml` instead of Google Doc viewer for PDF files.

Reply:
Great suggestion – done!